src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -117,6 +126,10 @@ export default class Pilot {
     maintainAltitude(currentAltitude, altitude, expedite, shouldUseSoftCeiling, airportModel) {
         const { minAssignableAltitude, maxAssignableAltitude } = airportModel;
         let clampedAltitude = clamp(minAssignableAltitude, altitude, maxAssignableAltitude);
+        
+        if (altitude > maxAssignableAltitude) {
+            this.assignedUnattainableAltitude = false;
```

Comment:
two things here:
1) This property seems to be used only within this method, thus, it _should not_ be an instance variable. This should be declared and used only within this method.
1) boolean type properties should be prefixed with the word `is` and worded in a truthy manner. so this could be revised to `isAssignedAltitudeAttainable`